setup.py

```diff
@@ -12,5 +17,6 @@
   author='balloob, ggravlingen',
   author_email='no@email.com',
   url='https://github.com/ggravlingen/pytradfri',
-  download_url=DOWNLOAD_URL
+  download_url=DOWNLOAD_URL,
+  install_requires=REQUIRES,
```

Comment:
Since we depend on a branch of aiocoap to be installed, should we actually depend on it in our setup.py ? We wouldn't want that version to be installed right ?